cljfmt/src/cljfmt/core.cljc

```diff
@@ -213,17 +235,27 @@
   (and (> depth 0)
        (= (inc idx) (index-of (nth (iterate z/up zloc) depth)))))
 
-(defn- fully-qualify-symbol [possible-sym alias-map]
+(def ^:private named? (some-fn string? keyword? symbol?))
+
+(defn- fully-qualify-symbol [possible-sym zloc alias-map]
```

Comment:
This should likely be just: `[zloc alias-map]`.

Reply:
Sorry, I can't follow. The `fully-qualify-symbol` naming suggest the defn is about processing a symbol.

In any case I can't imagine what kind of change is expected...

Reply:
In all instances of `fully-qualify-symbol`, we're writing:

```clojure
(fully-qualify-symbol (form-symbol zloc) zloc alias-map)
```

Instead let's just pass the zloc of the symbol:

```clojure
(fully-qualified-symbol zloc alias-map)
```

Reply:
One of the occurrences is like this:

```clojure
                     (fully-qualify-symbol (form-symbol top)
                                                        zloc
                                                        alias-map)
```

`top` being different than `zloc`

Reply:
Yeah, but it could easily just be rewritten using `top` in both locations:

```clojure
(fully-qualify-symbol (form-symbol top) top alias-map)
```

Unless I'm wrong (again!), it doesn't matter where the zloc argument is, because we're always going up to the top level.

Reply:
In:

```clojure
(defn- inner-indent [zloc key depth idx alias-map]
  (let [top (nth (iterate z/up zloc) depth)]
    (if (and (or (indent-matches? key
                                  (fully-qualify-symbol (form-symbol top)
                                                        zloc
                                                        alias-map))
                 (indent-matches? key
                                  (remove-namespace (form-symbol top))))
             (or (nil? idx) (index-matches-top-argument? zloc depth idx)))
      (let [zup (z/up zloc)]
        (+ (margin zup) (indent-width zup))))))
```

...the thing called `top` is not the absolute top, but the thing up `depth` levels.

`fully-qualify-symbol` cannot guess that depth.

Reply:
It doesn't need to guess it. `find-namespace` will work with any `zloc`. So:

```clojure
(fully-qualify-symbol (form-symbol top) zloc alias-map)
```

Will produce the same result as:

```clojure
(fully-qualify-symbol (form-symbol top) top alias-map)
```

Reply:
Got it! Done.